layouts/shortcodes/observability_pipelines/configure_existing_pipelines/source_env_vars/http_server.md

```diff
@@ -1 +1,3 @@
-REUSE INSTRUCTIONS
\ No newline at end of file
+HTTP/S server address.
+- The Observability Pipelines Worker listens to this socket address for logs from the HTTP/S server forwarder. For example, `0.0.0.0:9997`.
```

Comment:
```suggestion
- The Observability Pipelines Worker listens to this socket address for logs from the HTTP/S server forwarder. For example, `0.0.0.0:9997`.
```
I usually try to avoid "example" sentence fragments but I don't see a good way to restructure this example without moving it up to the "HTTP/S server address" line. If you don't want to do that, I think it's fine to leave it!